### DISCUSSION TITLE
Combat animations or blood compatibility?

### DISCUSSION THREAD

**Discussion by tinwe** [fvtt-fria-ligan/twilight2000-foundry-vtt#70] [Ideas]
Is there a way, or any plans, to make this module compatible with the Automatic combat Animation or perhaps some added blood and gore to gain some visual feedback for the attacks?

**Comment by Stefouch** [comment]
Do you have a link to this module so I can check if it's feasible ?

**Reply by tinwe** [reply]
> Do you have a link to this module so I can check if it's feasible ?

Adding combat animations such as gun fire, through the "Automated Animations" pluging, might prove a little difficult, but I would really hope for support for the Blood 'n Guts module.
https://foundryvtt.com/packages/blood-n-guts
On the other hand, that module doesn't seem to have been updated for half a year, despite that Foundry page claims that it was last updated a monthy ago, and still doesn't work with Foundry 0.9 yet :(
https://github.com/edzillion/blood-n-guts/issues/292
